langserver/initialize.go

```diff
@@ -41,7 +41,7 @@ func (h *Handler) handleInitialize(ctx context.Context, conn *jsonrpc2.Conn, req
 			HoverProvider:              true,
 			CodeActionProvider:         true,
 			CompletionProvider: &lsp.CompletionOptions{
-				ResolveProvider:   true,
+				ResolveProvider:   false,
```

Comment:
I found the cause of the issue. :cry:

Reply:
I seeeeee.

thanks, now vscode shows no error 👍